src/mmw/js/src/data_catalog/controllers.js

```diff
@@ -14,10 +14,32 @@ var DataCatalogController = {
             'was_data_catalog_visible': true,
         });
 
-        var searchModel = new models.DataCatalogModel();
-        var resources = new models.ResourceCollection();
+        var form = new models.SearchForm();
 
-        var view = new views.DataCatalogWindow(searchModel, resources);
+        var catalogs = new models.Catalogs([
+            new models.Catalog({
```

Comment:
Mostly to reduce repetition in the code.

```js
var catalogs = new Catalogs([new Catalog({}), new Catalog({}), new Catalog({})]);
```

We've used it previously [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/analyze/models.js#L71).

But I don't have a hard preference. Type hints are valuable wherever we can get them, and if the formatting favors them then they should be kept.